streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -87,7 +87,8 @@ class FileInputDStream[K, V, F <: NewInputFormat[K, V]](
    * files are visible within this window, then the file will get selected in the next batch.
    */
   private val minRememberDurationS =
-    Seconds(ssc.conf.getTimeAsSeconds("spark.streaming.minRememberDuration", "60s"))
+    Seconds(ssc.conf.getTimeAsSeconds("spark.streaming.fileStream.minRememberDuration",
+    ssc.conf.get("spark.streaming.minRememberDuration", "60s")))
```

Comment:
The second line is a cotinuation of the first line, so has to be further indented. Since its making this val multiline, please put a `{` `}` around it.